configuration/configuration.go

```diff
@@ -102,16 +102,18 @@ const (
 	varNotificationServiceURL   = "notification.serviceurl"
 )
 
-// ConfigurationData encapsulates the Viper configuration object which stores the configuration data in-memory.
-type ConfigurationData struct {
+// Registry encapsulates the Viper configuration registry which stores the
+// configuration data in-memory.
+type Registry struct {
 	v               *viper.Viper
 	tokenPublicKey  *rsa.PublicKey
 	tokenPrivateKey *rsa.PrivateKey
 }
 
-// NewConfigurationData creates a configuration reader object using a configurable configuration file path
-func NewConfigurationData(configFilePath string) (*ConfigurationData, error) {
-	c := ConfigurationData{
+// NewRegistry creates a configuration reader object using a configurable
+// configuration file path.
+func NewRegistry(configFilePath string) (*Registry, error) {
```

Comment:
How about using just `New()`?

Reply:
As long as there's nothing else that has a `NewXXX` function in the `configuration` package, then yes.

Reply:
I see your point. There is another `NewXXX`. So, please ignore this.

Reply:
@baijum as discussed on MM, there's no other `NewXXX` in the configuration package. Therefore I've renamed `configuration.NewRegistry` to `configuration.New` and `configuration.GetRegistry` to `configuration.Get` in e59d72fb252632efaeb17ba21aedb55a12eeb800.